packages/jasmine/src/version_check.js

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+const pkg = require('./package.json');
```

Comment:
Looks like this file is generic so it can live in `//:tools/npm_version_check.js` or something like that and then copied into each of the packages that needs it with a genrule as a dep of the npm_package:

```
genrule(
    name = "npm_version_check",
    srcs = ["//:tools/npm_version_check.js"],
    outs = [":npm_version_check.js"],
    cmd = "cp $< $@",
)
```

and then just add `:npm_version_check` to the deps of the npm_package for that package.